src/TestEdgeAnalysis.jl

```diff
@@ -0,0 +1,65 @@
+using Test
+using DiffEqBase
+using Sundials
+
+@testset "Species Selection" begin
+    outcore = readinput("../src/testing/minimal.rms")
+    outedge = readinput("../src/testing/minimal_edge.rms")
+    corespcs = outcore["phase"]["Species"]
+    edgeishspcs = outedge["phase"]["Species"]
```

Comment:
Why `edgeishspcs`? Is this to deal with cases that has species both exist in core and edge input files?

Reply:
Yep, it isn't just the edge species.

Reply:
Thanks! This is very useful